lib/kitchen/directions/bake_generic_eoc_section/v1.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeGenericEocSection
```

Comment:
What might also help re the "what does `klass` mean?" comment above... should the name of this direction be different?  

The current name sounds like it is making a new section at the end of a chapter.  But I don't think that's it.  More that it is moving sections to a new container at the end of a chapter.  Would `MoveSectionsToEoc` be better? (or `MoveSectionsToNewEocContainer`) The direction would still yield sections allowing them to be baked.  Then maybe `klass` becomes `section_class` and then readers will be like "oh ok this variable is the class of the sections I want moved to the end".

(I see below that `klass` is used for multiple jobs which might make you not want to change it, but I think this one field being used for 4 different purposes only one of which is as a CSS class is probably a problem)

Reply:
That makes sense -- by 4 different purposes, do you mean the title lookup & the middle part of the wrapper class? I can change the variable name (`klass` -> `section_class`) here and in the other eoc sections.

As for the direction name for `BakeGenericEocSection`, what do you think about `MoveCustomSectionToEocContainer` to emphasize that it differs from other more specific directions in that it allows the dev to pass in more custom parameters/modify the yielded sections?

Reply:
`Custom` is fine.